aspnetcore/toc.md

```diff
@@ -77,6 +77,7 @@
 ### [🔧 HTML helpers](mvc/views/html-helpers.md)
 ### [Tag helpers](mvc/views/tag-helpers/intro.md)
 #### [Authoring tag helpers](mvc/views/tag-helpers/authoring.md)
+#### [Tag Helpers working with forms](mvc/views/working-with-forms.md)
```

Comment:
Or "Using tag helpers in forms" or "Using tag helpers to work with forms"

Also change the title/H1 in the doc.

Consider removing the TOC entry from under Controllers.  It can be in both places but the TOC currently doesn't handle those cases well so it's best to avoid duplicating nodes if we can.